hydrolysis-analyzer.html

```diff
@@ -91,6 +91,7 @@
       var options = {
         clean:  this.clean,
         filter: this.transitive ? function() { return false; } : null,
+        attachAst: true,
```

Comment:
It hints hydrolysis to add AST references to a variety of places. I am convinced that there aren't any use cases that would want this off, now that I've tossed naive JSON serializability out of the window.